docs/pages/helm/guides/gcp.mdx

```diff
@@ -0,0 +1,313 @@
+---
+title: Running a highly-available Teleport cluster using GCP, GKE and Helm
+description: How to install and configure a highly-available Gravitational Teleport cluster using Google Cloud Platform and Google Kubernetes Engine
+---
+
+In this guide, we'll go through how to set up a highly-available Teleport cluster with multiple replicas in Kubernetes
+using Teleport Helm charts and Google Cloud Platform products (Firestore and Google Cloud Storage).
+
+This guide assumes that you've finished the [Getting Started with the Teleport Helm chart repository guide](./getting-started.mdx) and have Helm configured successfully.
+
+If not, please finish that guide first before starting this one!
+
+### Setting up an HA Teleport cluster using Helm
+
+In `gcp` mode, the `teleport-cluster` Helm chart uses Google Cloud Firestore for storing its backend database and audit logs. The highly-available
+shared storage of Google Cloud Firestore means that multiple Teleport pods can use the same backing database, enabling a Teleport cluster to scale
+efficiently with load.
+
+The chart also uses Google Cloud Firestore to hold Teleport's audit log, keeping track of actions performed within your cluster.
+
+Teleport's session recordings will be stored in a Google Cloud storage bucket, providing functional, efficient storage for any number of users.
+
+### Mandatory settings
+
+You'll need five pieces of information to configure `gcp` mode:
+
+| Key | Description | Example |
```

Comment:
Same here, let's move the mandatory settings to reference, and you are already enumerating the mandatory settings in the charts values and args.